mesonbuild/scripts/coverage.py

```diff
@@ -56,10 +52,11 @@ def coverage(source_root, build_root, log_dir):
                                '-a', initial_tracefile,
                                '-a', run_tracefile,
                                '-o', covinfo])
-        remove_dir_from_trace(lcov_exe, covinfo, '/usr/include/*')
-        remove_dir_from_trace(lcov_exe, covinfo, '/usr/local/include/*')
-        remove_dir_from_trace(lcov_exe, covinfo, '/usr/src/*')
-        remove_dir_from_trace(lcov_exe, covinfo, '/usr/lib/llvm-*/include/*')
+        # Remove all directories outside the source_root from the covinfo
+        subprocess.check_call([lcov_exe,
+                               '--extract', raw_tracefile, 
```

Comment:
__[Flake8]__

[W291] trailing whitespace


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Whoops. First time contributor, should I amend my commit and force push a new version to my fork?

Reply:
Yes.